package.json

```diff
@@ -44,8 +44,10 @@
     "@babel/core": "^7.22.9",
     "@babel/eslint-parser": "^7.22.9",
     "@babel/preset-env": "^7.22.9",
+    "@symfony/webpack-encore": "^4.4.0",
     "autoprefixer": "^10.4.14",
     "babel-loader": "^9.1.3",
+    "core-js": "^3.32.0",
```

Comment:
to match the Webpack Encore Bundle recipe, see https://github.com/symfony/recipes/blob/main/symfony/webpack-encore-bundle/1.10/package.json#L8